vllm/engine/arg_utils.py

```diff
@@ -382,6 +382,7 @@ def add_cli_args(parser: FlexibleArgumentParser) -> FlexibleArgumentParser:
             'Backend-sepcific options can be supplied in a comma-separated '
             'list following a colon after the backend name. Valid backends and '
             'all available options are: [xgrammar:no-fallback, '
+            'xgrammar:disable_any_whitespace, '
```

Comment:
We need to standardize on either `-` or `_` here. You could convert them to either form internally, but externally we should be consistent

Reply:
Whoops, I set a precedent 😅

Reply:
You chose correctly though!